src/main/resources/application.properties

```diff
@@ -4,4 +4,7 @@ spring.datasource.password=Entermail13*
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
+server.servlet.context-path=/api
 
+jwt.expiration=360000
+jwt.secret=veryveryveryveryveryveryverysecretkey
```

Comment:
It’s a project for your CV, so better use a more secure secret 🤫

Reply:
changed